README.md

```diff
@@ -182,7 +182,13 @@ vim.api.nvim_set_keymap(
 )
 ```
 
-Alternatively, you can also access the picker as a function via `require "telescope".extensions.file_browser.file_browser` natively in lua.
+Alternatively, you can also access the picker as a function via `require "telescope".extensions.file_browser.file_browser()` natively in lua as follows:
```

Comment:
Maybe while we're at it, we can standardize the rest of this keymap section to use `vim.keymap.set` and put it all under one codeblock.

```lua
vim.keymap.set("n", "<space>fb", ":Telescope file_browser<CR>")

-- open file_browser with the path of the current buffer
vim.keymap.set("n", "<space>fb", ":Telescope file_browser path=%:p:h select_buffer=true<CR>")

-- Alternatively, using lua API
vim.keymap.set("n", "<space>fb", function()
	require("telescope").extensions.file_browser.file_browser()
end)
```

Reply:
I've standardized keymaps as you requested